packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -103,19 +109,17 @@ export async function makeRequestOptions(
 	} else if (includeCredentials === undefined) {
 		credentials = "same-origin";
 	}
-
 	const info: RequestInit = {
 		headers,
 		method: "POST",
 		body: binary
 			? args.data
 			: JSON.stringify({
-					...otherArgs,
-					options: options && otherOptions,
```

Comment:
Why this change?

Reply:
while doing custom streamRequests for Mistral or OpenAI, they'll refuse requests with that extra `options` params, so the idea is only to try to send all the extra params only for HF task requests, wdyt? For instance, it seems like the base params `model, stream, messages` is often accepted, then each API has their specific `otherArgs` which is nice to send as well

Reply:
I could add some of these failing tests if you think is useful. IMO we should support well our new messages API and make sure if works the basic for other APIs, however it's mainly important to work with TGI